src/components/Meals/Meals.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import { Carousel, CarouselItem } from 'react-bootstrap';
+import MealCard from '../MealCard/MealCard';
+import '../Meals/meals.css';
+
+export default class Meals extends Component {
+  render() {
+    return (
+      <div>
+        <div className="available-meals">
+          <h1 className="meals-title">Available Meals Right Now</h1>
+          <p className="meals-quote">
+            `If you dont try this app, you wont become the hero you were meant
+            to be`
+          </p>
+          <Carousel>
+            <CarouselItem>
+              <div className="items">
+                <MealCard
```

Comment:
i want this to me map array / object so we map it this will help when we reach the back end